db/schema.rb

```diff
@@ -366,7 +366,6 @@
     t.string "slug"
     t.tsvector "searchable"
     t.string "lis_result_sourced_id"
-    t.string "version", default: "1.0", null: false
```

Comment:
I haven't made any changes to the schema.rb, not really understanding why its listed in files changed...

Reply:
have tried to revert changes again